gatling/user-files/simulations/WsBulkImport.scala

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright 2011-2018 GatlingCorp (https://gatling.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package computerdatabase
+
+import io.gatling.core.Predef._
+import io.gatling.http.Predef._
+import scala.concurrent.duration._
+
+class WsBulkImportDocument extends Simulation {
+  val host = System.getProperty("host", "localhost")
+  val requests = System.getProperty("requests", "2000").toInt
+  val users = System.getProperty("users", "1").toInt
+  val duration = System.getProperty("duration", "1").toInt
+
+   val bulkData = """ 
+   {
+	"bulkData": [{
+    "index": {
+      "_index": "nyc-open-data",
+      "_type": "yellow-taxi"
+    }
+  },
+  {
+    "name": "FCA-Ardepharm",
+    "radius": 25,
+    "location": {
+      "lat": 45.045626,
+      "lon": 4.846281
+    }
+  }]
+}
+  """
+
+  val query = """
+    {
+      "index": "nyc-open-data",
+      "collection": "yellow-taxi",
+      "controller": "bulk",
+      "action": "import",
+      
+      "body": """ + bulkData + """
+    }
+  """
+
+  val httpProtocol = http
+    .baseUrl("http://" + host + ":7512")
+    .acceptHeader("text/html,application/json,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8")
+    .acceptEncodingHeader("gzip, deflate")
+    .userAgentHeader("Gatling2")
+    .wsBaseUrl("ws://" + host + ":7512")
```

Comment:
```suggestion
    .wsBaseUrl(s"ws://${host}:7512")
```

Reply:
I won't report other similar nitpickings, I let you choose if you want to replace string concatenation with interpolations to make it more readable.